Added ceremonies to all medicine cat traits

hobohime opened the pull request:
All traits for medicine cats now have a ceremony attached to them!

hobohime committed:
Added ceremonies to all medicine cat traits

Please review 'wise' closely, I'm unsure if we should add it, but it is fun. Canon warrior cats does have reincarnation but it may be unsuitable for clangen, so I would like feedback, and am pointing it out so it doesn't get added without review.

Ryos00 commented on resources/dicts/events/ceremonies/ceremony_text.json:
```diff
@@ -800,38 +800,91 @@
-            "daring": [],
-            "empathetic": [],
+            "calm": [
+                "(prefix)paw has proven themsleves with one of the most invaluble traits a medicine cat can have, a level head under pressure. The stars of Silverpelt twinkle in joy and acceptance as m_c recives their full name and duites as a full medicine cat."
```
I would use a hyphen here instead of a comma
"...a medicine cat can have - a level head under pressure."
also duites

capillarys replied:
i also think a semi-colon would be suitable, "... a medicine cat can have: a level head under pressure."

Ryos00 commented on resources/dicts/events/ceremonies/ceremony_text.json:
```diff
@@ -800,38 +800,91 @@
+                "(prefix)paw's charming attitude does as much for their patients as their skill does. They will continue to heal with words and herbs as m_c now that they are a full medicine cat of c_n."
+            ],
+            "childish": [
+                "Though their actions may come off as lighthearted, (prefix)paw has worked hard to earn their name of m_c, and mews that they'll protect c_n as a medicine cat. Though they start giggling right after they say it, causing a few purrs of amusement from onlookers both here, and here after."
```
the last bit of this reads a little weird to me. "both here, and here after." I'm not sure what this means

hobohime replied:
"here after" refers to the afterlife, I was making a pun but I can change it

Ryos00 commented on resources/dicts/events/ceremonies/ceremony_text.json:
```diff
@@ -800,38 +800,91 @@
+                "No one knows what (prefix)paw is thinking as they recive their medicine cat name of m_c, only that their eyes are focused, clear, and as immovable as ice. No one doubts that they will be well suited to the heartbreaks that wait for all medicine cats."
+            ],
+            "compassionate": [
+                "Excellent medicine cats have as much heart as they do brains, and (prefix)paw proves they have both by the time they earn thier name of m_c, proving a gentle touch does as much good as the right herbs."
```
*their

Ryos00 commented on resources/dicts/events/ceremonies/ceremony_text.json:
```diff
@@ -800,38 +800,91 @@
+                "No one knows why (prefix)paw brought a mossball to their medicine cat naming ceremony, but StarClan grants them their name of m_c all the same, welcoming them into the full mysteries of StarClan."
+            ],
+            "strict": [
+                "The way (prefix)paw is sitting is so stiff you could use them as reenforcement of the camp walls. Even when they get their medicine cat name of m_c, they don't loosen up, and only seem to sit up even more."
```
*reinforcement

Ryos00 commented on resources/dicts/events/ceremonies/ceremony_text.json:
```diff
@@ -800,38 +800,91 @@
+                "The other medicine cats mistake it as being calm, but (prefex)paw isn't quiet because they're at peace, but because they are angry. Every snide remark, every time someone sneered at them, every time they were treated less than, all of it goes through their mind. Now that they are m_c, c_n better treat them with respect, or there would be consequences."
+            ],
+            "wise": [
+                "Medicine cat training had just felt right for (prefix)paw, more like remembering then learning, like they had done this before. When it comes time for their naming, they ask to name themselves, and chose m_c for their name. It feels right for them to be called m_c."
```
I like it! Just vague enough to not be specifically about reincarnation, but suggesting it

hobohime committed:
Changed typos and one vague wording

"here after" is now "StarClan" to clarify the meaning. Various typo corrections.
